src/com/google/enterprise/adaptor/fs/FsAdaptor.java

```diff
@@ -1086,81 +1093,61 @@ private void getFileAcls(Path doc, Response resp) throws IOException {
     }
   }
 
-  /* Makes HTML document with web links to namespace's DFS links. */
-  private void getDfsNamespaceContent(Path doc, DocId docid, Response resp)
-      throws IOException {
-    // TODO (bmj): Handle large Namespaces by paying attention to maxHtmlLinks
-    // and sending excess to DocIdPusher.
-    resp.setNoIndex(!indexFolders);
-    try (HtmlResponseWriter writer = createHtmlResponseWriter(resp)) {
-      writer.start(docid, getFileName(doc));
-      try (DirectoryStream<Path> links = delegate.newDfsLinkStream(doc)) {
-        for (Path link : links) {
-          DocId docId;
-          try {
-            docId = delegate.newDocId(link);
-          } catch (IllegalArgumentException e) {
```

Comment:
You're removing this. Is it just an impossible exception? There's another one of them in validateStartPath.